sdks/python/container/build.gradle

```diff
@@ -59,7 +59,15 @@ golang {
 
 docker {
   name containerImageName(name: "python")
-  files "./build/"
+  files "./build"
+  // If the user has specified additional python requirements, copy them otherwise make an empty file
+  if (project.hasProperty("python_requirements")) {
```

Comment:
Suggestion: instead of making the current python Dockerfile more involved, perhaps it would be preferable to add a 2nd Dockerfile and target FROM an existing image. Then users who just want to pre-install various dependencies can extend even released images. It would be faster and more robust than rebuilding the whole thing.

Reply:
That's a really good point, I forgot about the users who might not building from source.